illumio_connector.py

```diff
@@ -543,7 +549,7 @@ def check_starttime_greater_than_endtime(self, datetime_start, datetime_end):
 
     def connect_pce(self, action_result):
         """
-        Connects to the PCE server.
+        Connect the PCE server.
```

Comment:
If you want to change from descriptive to imperative to align with PEP 257 that's fine, but removing `to` here and `the` above are incorrect.

Reply:
Done